### PR TITLE
updating scripts to be compatible with latest numpy versions

### PR DESCRIPTION
**Pull request by alperyeg**
PR to fix breaking travis script:
* updated to latest numpy, scipy and pandas versions in the corresponding env build
* updated several scripts which broke because of the new libraries (incorporating PR #168)
* updated accompanying tests 
* pep8 in those scripts

### CODE REVIEW

File: elephant/asset.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def _signals_same_tstart(signals):
     signals : list
         a list of signals (e.g. AnalogSignals or SpikeTrains) having
         attribute `t_start`
-
+en
```
Comment (Junji110) [inline elephant/asset.py]: What is this?

Comment (alperyeg) [reply elephant/asset.py]: It's a typo. If you look at my second commit it should have been removed. Please squash when merging.

Comment (Junji110) [reply elephant/asset.py]: Ah, upper commit is the later one, that I didn't notice.
OK, I have no more comments now.

**Comment by Junji110** [comment]
Sorry, I pressed the revert button by mistake.
How can I cancel this?

**Comment by alperyeg** [comment]
I just checked out the master branch the commits are applied. I think we do not need to take action.

**Comment by Junji110** [comment]
The master branch is OK, but the revert created a new branch named "revert-173-fix/travis_numpy" in this repository.
It shouldn't exist.
Can you delete this branch?

**Pull request merged**